crates/ruma-client-api/src/future/send_future_message_event.rs

```diff
@@ -52,7 +52,7 @@ pub mod unstable {
 
         /// Additional parameters to describe sending a future.
         ///
-        /// Only three combinations for `future_timeout` and `future_group_id` are possible.
+        /// Only three combinations for `future_timeout` and `future_parent_id` are possible.
         /// The enum [`FutureParameters`] enforces this.
         #[ruma_api(query_all)]
         pub future_parameters: FutureParameters,
```

Comment:
I am not sure why we keep using the enum since there is only one query parameter possible currently.

Reply:
I edited the doc comment on the enum explaining the situation. I think its also good if the enum currently on supports the one possible parameter.
There are still discussions about adding "Action" delayed events (futures).